.env.example

```diff
@@ -37,7 +37,7 @@ WS_URL=ws://localhost:8250/ws
 # llm
 ANSWER_AGENT_LLM="mistral"
 INTENT_AGENT_LLM="openai"
-REPORT_AGENT_LLM="mistral"
```

Comment:
why get Diana to implement Mistral and then just change it back again?

Reply:
We want the report agent on openai because the file processing is more advanced. We want Mistral to be able to handle "chat_with_file" method so that if someone did have it on Mistral the app wouldn't crash

Reply:
It seemed more complete this way, but that's fair, we could have split that work into a tech-debt ticket instead